components/eventing-controller/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kyma-project/kyma/components/eventing-controller
 
-go 1.16
+go 1.17
```

Comment:
please run 
`go mod tidy` and update the pr. switching to go 1.17 semantics introduced require directives for indirect dependencies to be added to the go.mod file

Reply:
done